test/e2e/framework/util.go

```diff
@@ -4491,7 +4479,34 @@ func CheckConnectivityToHost(f *Framework, nodeName, podName, host string, timeo
 			Logf("pod %s/%s logs:\n%s", f.Namespace.Name, pod.Name, logs)
 		}
 	}
+	return err
+}
 
+// CheckConnectivityToHost launches a pod to test connectivity to the specified
+// host. An error will be returned if the host is not reachable from the pod.
+//
+// An empty nodeName will use the scheduler to choose where the pod is executed.
+func CheckConnectivityToHost(f *Framework, nodeName, host string, timeout int) error {
+
+	podName := "ping-test"
+	cmdPing := []string{
+		"ping",
+		"-c", "3", // send 3 pings
+		"-W", "2", // wait at most 2 seconds for a reply
+		"-w", strconv.Itoa(timeout),
+		host,
+	}
+	err := checkConnectivityWithCommand(f, nodeName, podName, cmdPing)
```

Comment:
@jdumars This is the issue that is being tracked.

Reply:
ack